code/game/machinery/dance_machine.dm

```diff
@@ -166,10 +166,10 @@
 				volume = 0
 				return TRUE
 			else if(new_volume == "max")
-				volume = 100
+				volume = initial(volume)
 				return TRUE
 			else if(text2num(new_volume) != null)
-				volume = text2num(new_volume)
+				volume = (text2num(new_volume))
```

Comment:
remove the parens now that you dont need it anymore

Reply:
good eye